README.md

```diff
@@ -47,6 +47,27 @@ To run this sample, you'll need:
 - A user account in your Microsoft Entra tenant. This sample will not work with a Microsoft account (formerly Windows Live account). Therefore, if you signed in to the [Microsoft Entra admin center](https://entra.microsoft.com) with a Microsoft account and have never created a user account in your directory before, you need to do that now.
 - Setup [MAUI](https://learn.microsoft.com/dotnet/maui/get-started/installation?tabs=vswin&view=net-maui-7.0). This will require Visual Studio on PC, and on a Mac Machine.
 
+To run the sample on VS Code, you'll need:
+
+- For iOS/Mac, A specific version of Xcode, which depends on the version of .NET MAUI that you're using is needed. For information, see [Release Versions](https://github.com/dotnet/maui/wiki/Release-Versions), make sure to install the iOS simulations during the xcode installation.
+- [VS Code](https://code.visualstudio.com/download). Install or update Visual Studio Code.
+  - In Visual Studio Code, in the Extensions tab, search for ".NET MAUI" and install the .NET MAUI extension. The .NET MAUI extension automatically installs the C# Dev Kit and C# extensions, which are required for the .NET MAUI extension to run.
+- Install [.NET](https://learn.microsoft.com/en-us/dotnet/core/install/windows)
+- Install the .NET MAUI workload.
+  - Windows - dotnet workload install maui
+  - Mac - sudo dotnet workload install maui
+- While trying to build and launch the MAUI project on Mac, if you run into "Please Verify the Xcode Installation" Error, do the following:
+  - In your Terminal, first run:
+    $ /Applications/Xcode.app/Contents/Developer/usr/bin/simctl delete unavailable
+  - Reboot, then reopen your MAUI solution
+- For iOS/Mac, setup provisioning on your device by following the below steps:
+  - Open Xcode -> Settings -> Accounts, Add your Microsoft Apple ID
+  - Click on "Download Manual Profiles" and wait for it to finish
```

Comment:
Also add dowloading certificates in addition of downloading manual profiles.

Reply:
updated